### PR TITLE
Issue #34: News Feed Page

### PR DESCRIPTION
**Pull request by cottonswab**
- Create "feed" component
- Send fetch request to get most recent photo from each user followed (backend api loop (get user.following.photos.photo_n))
- Front end display results

### CODE REVIEW

File: client/src/pages/homepage.js [hunk 1]
```diff
@@ -186,11 +212,11 @@ class HomePage extends Component {
   //             }}
```
Comment (loganmcmullen) [inline client/src/pages/homepage.js:212]: delete this commented out stuff of use /* ------- */

File: client/src/pages/homepage.js [hunk 2]
```diff
@@ -186,11 +212,11 @@ class HomePage extends Component {
...
   //           />
   //         </li>
-  //       ))} */}
+  //       ))} *///}
   //         </Col>
   //       </Row>
   //     </Container>
-
+/*
```
Comment (loganmcmullen) [inline client/src/pages/homepage.js:219]: remove some of the commented stuff

File: client/src/components/displayHomepage.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+            });
+    }
+    componentDidMount() {
+        this.loadImages();
+    }
+    render() {
+        return (
+            <Container>
+                <Row>
+                    {this.state.img.map(item => {
```
Comment (loganmcmullen) [inline client/src/components/displayHomepage.js:72]: I see you used the template from the profile pictures which is great. Good way to save time and keep consistency throughout

**Review by loganmcmullen — APPROVED** [review]

File: client/src/pages/homepage.js (hunk 2 above)
Comment (cottonswab) [reply client/src/pages/homepage.js:219]: Will do.

**Review by gregtuc — CHANGES_REQUESTED** [review]
There needs to be an ability to see who posted the photo. I recommend making another state variable and adding the name of the user who's photo it is to the Card. It would also be better if the cards were laid out vertically in a single column, not in Profile-page format. After this is done the request can be pulled in

File: client/src/components/displayHomepage.js [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+                                    border="dark"
+                                >
+                                    <Card.Img
+                                        variant="top"
+                                        src={"http://localhost:8000/image/" + item.image}
+                                        alt="Image could not be loaded"
+                                    />
+                                    <Card.Body>
+                                        <Card.Title>{item.description}</Card.Title>
+                                        <Card.Text>This will be the comment section</Card.Text>
```
Comment (ginywiny) [inline client/src/components/displayHomepage.js:89]: These sections will be updated with the comments section that I am still in the process of completing.

**Review by ginywiny — APPROVED** [review]

**Review by gregtuc — APPROVED** [review]
Perfect. Thanks for making the changes. Looks good

**Pull request merged**